### PR TITLE
Change failure assertion during setIdentityDkimEnabled

### PR DESCRIPTION
**Pull request by palpatim**
*Issue #, if available:*
N/A

*Description of changes:*
It's unclear why the test would have started failing recently, but the logic of the test, as edited, seems sound to me. I'm verifying that assumption with the SES team and will merge this PR once I have that verification. In the meantime, I'm submitting this PR for review to make sure that others agree with my assessment.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-android-sdk-ses-test/src/androidTest/java/com/amazonaws/services/email/IdentityIntegrationTest.java [hunk 1]
```diff
@@ -344,23 +347,17 @@ public void testDkim() throws Exception {
...
-                for (String token2 : dkim.getDkimTokens()) {
-                    if (token1.equals(token2)) {
-                        found = true;
-                        break;
-                    }
-                }
-                assertTrue(found);
+            List verifyDomainResultTokens = dkim.getDkimTokens();
+            for (String token : attributes.getDkimTokens()) {
+                assertTrue(verifyDomainResultTokens.contains(token));
```
Comment (desokroshan) [inline aws-android-sdk-ses-test/src/androidTest/java/com/amazonaws/services/email/IdentityIntegrationTest.java]: This is so much more succinct. 
Minor suggestion: assertTrue(dkim.getDkimTokens().containsAll(attributes.getDkimTokens()))?

Comment (palpatim) [reply aws-android-sdk-ses-test/src/androidTest/java/com/amazonaws/services/email/IdentityIntegrationTest.java]: TIL Java List has a `containsAll` method.

Done.

**Comment by palpatim** [comment]
I verified with the service team that the behavior has changed to allow customers to enable DKIM signing before the identity is successfully verified.

Merging.

File: aws-android-sdk-ses-test/src/androidTest/java/com/amazonaws/services/email/IdentityIntegrationTest.java [hunk 2]
```diff
@@ -344,23 +347,15 @@ public void testDkim() throws Exception {
...
-                for (String token2 : dkim.getDkimTokens()) {
-                    if (token1.equals(token2)) {
-                        found = true;
-                        break;
-                    }
-                }
-                assertTrue(found);
-            }
+            List verifyDomainResultTokens = dkim.getDkimTokens();
+            assertTrue(verifyDomainResultTokens.containsAll(verifyDomainResultTokens));
```
Comment (desokroshan) [inline aws-android-sdk-ses-test/src/androidTest/java/com/amazonaws/services/email/IdentityIntegrationTest.java]: Shouldn't it be : 
```suggestion
            assertTrue(verifyDomainResultTokens.containsAll(attributes.getDkimTokens()));
```

Comment (palpatim) [reply aws-android-sdk-ses-test/src/androidTest/java/com/amazonaws/services/email/IdentityIntegrationTest.java]: Yes it should, thanks for catching!

**Pull request merged**